develop-docs/application/feature-flags/flagpole.mdx

```diff
@@ -176,8 +176,6 @@ features.add(
 feature.<feature_scope>:<feature_name>
 ```
 
-3. Add the feature name to the `flagpole.flagpole_only_features` list in sentry-options-automator's default [`app.yaml`](https://github.com/getsentry/sentry-options-automator/blob/483737d45dbc68253e926c3f860b5ae33111697b/options/default/app.yaml#L219) file, omitting the `feature.` prefix.
```

Comment:
We need to update lines 163 and the flagpole feature testing piece below ~183-215 mentioning testing with the flagpole_only_features stuff, otherwise this looks good!